ratpack-test/src/main/java/ratpack/test/internal/BlockingHttpClient.java

```diff
@@ -46,6 +48,7 @@ public ReceivedResponse request(HttpClient httpClient, URI uri, ExecController e
     AtomicReference<ExecResult<ReceivedResponse>> result = new AtomicReference<>();
 
     execController.fork()
+      .register(r -> r.add(AddressResolverGroup.class, DefaultAddressResolverGroup.INSTANCE))
```

Comment:
This does the necessary registry add so that the `HttpClient` works, but could be removed if we a more holistic approach to this.